ProjectLighthouse.Servers.GameServer/Controllers/Slots/ScoreController.cs

```diff
@@ -199,15 +210,23 @@ private string getScores
         int pageStart = -1,
         int pageSize = 5,
         string rootName = "scores",
-        string[]? playerIds = null
+        string[]? playerIds = null,
+        int? advId = null
     )
     {
 
+        
+        IQueryable<Score> scores;
+        if (advId != null) {
+            scores = this.database.Scores.Where(s => s.SlotId == slotId && s.Type == type && s.AdvSlotId == advId);
+        }
+        else {
+            scores = this.database.Scores.Where(s => s.SlotId == slotId && s.Type == type);
+        }
+
         // This is hella ugly but it technically assigns the proper rank to a score
         // var needed for Anonymous type returned from SELECT
```

Comment:
```suggestion
        // This is hella ugly but it technically assigns the proper rank to a score
        // var needed for Anonymous type returned from SELECT
        var rankedScores = this.database.Scores
            .Where(s => s.SlotId == slotId && s.Type == type)
            .Where(s => advId == null || s.AdvSlotId == advId)
            .AsEnumerable()
```
GitHub doesn't let me select line 229 but this can still be a one-liner